### ISSUE TITLE
Image load fails when path contains special chars (eg. "(" and ")")

### ISSUE THREAD

**Issue by ghost** [pisi/Reel#125]
on line 324 you should change:

``` js
images.length || t.css({ backgroundImage: url(opt.path + image) });
```

to

``` js
images.length || t.css({ backgroundImage: url('"' + opt.path + image + '"') });
```

This fixes the issue.

**Comment by ghost** [comment]
Path was like: http://myfancywebsite.com/images/folderwith(special)chars/imagename.jpg

**Comment by pisi** [comment]
Good point! Just verified this issue by test. But the fix will look a bit different than you suggest. You see the `url()` in:

``` js
backgroundImage: url(opt.path + image)
```

isn't actually the CSS `url()` just yet. This `url()` is a shared internal helper Javascript function which accepts an URL as a String, wraps the input with CSS `url()` and outputs the wrapped String which finally is used as the CSS declaration. This way the fix is needed only on one place and it will fix all its 4 occurrences within Reel :)

Thanks for reporting it!

**Comment by pisi** [comment]
Bugfix landed on the [development](/pisi/Reel/tree/development) branch. @dirkpennings, please verify this bug fix in your code.

**Comment by pisi** [comment]
This bugfix just landed in the [v1.2rc2](/pisi/Reel/tree/v1.2rc2) heading for the 1.2 launch. If you can, please verify this bugfix is working for you. Thanks!

**Comment by ghost** [comment]
Verified, it works! (tested on jQuery 1.6.2)

Thank you for the quick fix! Great plugin!
